src/tests/ftest/util/daos_utils.py

```diff
@@ -506,7 +507,18 @@ def container_get_prop(self, pool, cont):
         #   "error": null,
         #   "status": 0
         # }
-        return self._get_json_result(("container", "get-prop"), pool=pool, cont=cont)
+        props = None
+
+        if properties:
+            for prop in properties:
+                if props:
+                    props += ","
+                    props += prop
+                else:
+                    props = prop
```

Comment:
`str.join` is for exactly this :)
```suggestion
        props = ','.join(properties) if properties else None
```

Reply:
This is what I was looking for. Thanks!